### DISCUSSION TITLE
DeepL pro ver. API keys error from subtitle edit

### DISCUSSION THREAD

**Discussion by Leo93inseoul** [SubtitleEdit/subtitleedit#8446] [Q&A]
I subscribe DeepL pro ver.
I entered my DeepL API Key in the API KEY field, but the following notification massage appeared.
![1111](https://github.com/SubtitleEdit/subtitleedit/assets/171039283/34258133-4fde-47ea-bf07-1ad8a631e5ab)

how can i use my DeepL pro ver API keys?

**Comment by niksedk** [comment]
Works fine here... 

Try contacting DeepL support.
